gandalf.js

```diff
@@ -2,76 +2,77 @@
  * Extras Plugin
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?